common/build-style/void-cross.sh

```diff
@@ -0,0 +1,421 @@
+#
+# This helper is for void system crosstoolchain templates.
+#
+# Mandatory variables:
+#
+# - cross_triplet - the target triplet (e.g. aarch64-linux-gnu)
+# - cross_wordsize - 32 or 64
+# - cross_libc - glibc or musl
+# - cross_binutils_version
+# - cross_gcc_version
+# - cross_linux_version
+# - cross_linux_arch - the source ARCH of the kernel (e.g. arm64)
+# - cross_glibc_version
+# - cross_musl_version
+#
+# Optional variables:
+#
+# - cross_binutils_configure_args
+# - cross_gcc_bootstrap_configure_args
+# - cross_glibc_cflags
+# - cross_glibc_ldflags
+# - cross_glibc_configure_args
+# - cross_musl_cflags
+# - cross_musl_ldflags
+# - cross_musl_configure_args
```

Comment:
Not a big deal but I prefer to spell it as `cross_libc_{cflags,ldflags,configure_args}`

Reply:
i don't like that since it's inconsistent with the other subprojects being built